### PR TITLE
Abb 71

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Abb 71 and minor remarks in another one.

### CODE REVIEW

File: ParisBNF/abb/BNFabb71.xml [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+            <editor role="generalEditor" key="AB"/>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb71.xml:19]: remove `p` around the sentence

File: ParisBNF/abb/BNFabb71.xml [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+               </msContents>
+               <physDesc>
+                  <objectDesc form="Codex">
+                     <supportDesc>
+                        <support>
+                           <material key="parchment"/>
+                        </support>
+                        <extent>
+                           <measure unit="leaf" quantity="79">4+91+3</measure>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb71.xml:67]: something does not match. quantity 98? or 4+72+3?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb71.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb71.xml:67]: done

File: ParisBNF/abb/BNFabb71.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb71.xml:19]: ok

**Review by eu-genia — APPROVED** [review]

**Pull request merged**